docs/variants/talos_2/test-matrix.md

```diff
@@ -9,14 +9,17 @@ subjected from before the release of the new binary.
 
 | No.  | Supported test suite                              | Test suite ID | Supported test cases                 |
 |:-----|:--------------------------------------------------|:-------------:|:-------------------------------------|
-| 1.   | [coreboot base port][CBP]                         | CBP           | CBP001.001, CBP002.001, CBP004.001   |
+| 1.   | [Coreboot base port][CBP]                         | CBP           | CBP001.001, CBP002.001, CBP004.001   |
```

Comment:
`coreboot` should always be written in lower case.

Reply:
fixed